Formula/clickhouse-odbc.rb

```diff
@@ -0,0 +1,104 @@
+class ClickhouseOdbc < Formula
+  desc "Official ODBC driver implementation for accessing ClickHouse as a data source"
+  homepage "https://github.com/ClickHouse/clickhouse-odbc#readme"
+  url "https://github.com/ClickHouse/clickhouse-odbc.git",
+      tag:      "v1.1.10.20210822",
+      revision: "c7aaff6860e448acee523f5f7d3ee97862fd07d2"
+  license "Apache-2.0"
+  head "https://github.com/ClickHouse/clickhouse-odbc.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "icu4c"
+  depends_on "openssl@1.1"
+
+  on_macos do
+    depends_on "libiodbc"
+  end
+
+  on_linux do
+    depends_on "unixodbc"
+    depends_on "gcc" => :build
```

Comment:
Do you have C++ code? If you do you will need this at run time.

Reply:
No, at run time only `.so`'s or `.dylib`'s are used.

Reply:
Is there C++ code in your shared objects? If there are, you will still need GCC at runtime, especially if they link with GCC's `libstdc++`.

Reply:
Oh, I see what you mean. Yes, `libstdc++` will be a run-time dependency. Will change that.